src/index.tsx

```diff
@@ -1,15 +1,40 @@
 import './wdyr'
 import 'lib/polyfills'
 
+import * as Sentry from '@sentry/react'
 import { App } from 'App'
 import { AppProviders } from 'AppProviders'
+import { getConfig } from 'config'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
 import { renderConsoleArt } from 'lib/consoleArt'
 import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
+Sentry.init({
+  ...(window.location.hostname === 'localhost'
+    ? {}
+    : { dsn: getConfig().REACT_APP_SENTRY_DSN_URL }),
+  attachStacktrace: true,
+  denyUrls: ['alchemy.com'],
+  integrations: [
+    // Sentry.browserTracingIntegration(),
+    // Sentry.replayIntegration(),
+    Sentry.httpClientIntegration({
```

Comment:
Defaults to `[500, 599]` range which won't catch all our failures

Reply:
Hopefully we don't get rugged by something `429`ing us and burning our credits.

Reply:
That's a very valid point! We should probably exclude 429s? Though they *can* be useful to know when something wasn't working because of rate limits, I believe that we would rarely get value out of that and most of it should fall into expected failures.

Reply:
FYI: https://github.com/shapeshift/web/pull/6782/commits/6d7601c69e80f577c962d69af08d7f666b64fae8